src/wfs/src/main/java/org/geoserver/wfs/xml/filter/v1_1/PropertyNameTypeBinding.java

```diff
@@ -31,8 +31,8 @@
     NamespaceSupport namespaceSupport;
 
     public PropertyNameTypeBinding(
-            FilterFactory filterFactory, NamespaceSupport namespaceSupport, Catalog catalog) {
-        super(filterFactory);
+            FilterFactory2 filterFactory, NamespaceSupport namespaceSupport, Catalog catalog) {
+        super(filterFactory, namespaceSupport);
         this.namespaceSupport = namespaceSupport;
```

Comment:
The base class has the namespace support, use it instead of duplicating it. I've added a suggestion to make that field protected in the GeoTools PR.

Reply:
Done